application-security/admin_guide/book.yml

```diff
@@ -0,0 +1,204 @@
+    - name: GitLab
+      file: add-gitlab.adoc
+  - name: CI/CD Runs
+    dir: ci-cd-runs
+    topics:
+    - name: CI/CD Runs
+      file: ci-cd-runs.adoc
+    - name: AWS Code Build
+      file: add-aws-codebuild.adoc
+    - name: CircleCI
+      file: add-circleci.adoc
+    - name: GitHub Actions
+      file: add-github-actions.adoc
+    - name: Jenkins
+      file: add-jenkins.adoc
+    - name: Terraform Cloud (Sentinel)
+      file: add-terraform-cloud-sentinel.adoc
+    - name: Terraform Cloud (Run Tasks)
+      file: add-terraform-run-tasks.adoc
+    - name: Terraform Enterprise (Sentinel)
+      file: add-terraform-enterprise.adoc
+    - name: Terraform Enterprise (Run Tasks)
+      file: add-terraform-enterprise-run-tasks.adoc
+  - name: CI/CD Systems
+    dir: ci-cd-systems
+    topics:
+    - name: CI/CD Systems
+      file: ci-cd-systems.adoc
+    - name: Jenkins Plugins
+      file: add-jenkins-cicd-system.adoc
+    - name: CircleCI
+      file: add-circleci-cicd-system.adoc
+  - name: Package Registries
+    dir: package-registries
+    topics:
+    - name: Package Registries
+      file: package-registries.adoc
+    - name: Artifactory
+      file: add-private-registries.adoc
+  - name: Integrate IDE
+    dir: integrate-ide
+    topics:
+    - name: IDE
+      file: integrate-ide.adoc
+    - name: VS Code
+      file: connect-vscode.adoc
+    - name: IntelliJ
+      file: connect-intellij.adoc
+- name: Set Up a Network Tunnel (Transporter) to Connect Self-Hosted Repositories
+  file: setup-network-tunnel.adoc
+- name: Add Prisma Cloud Code Security Scanner as a Pre-Receive Hook
+  file: add-pre-receive-hooks.adoc
+- name: Code Repositories and Policy Management
+  dir: code-repositories-policy-management
+  topics:
+  - name: Code Repositories and Policy Management
+    file: code-repositories-policy-management.adoc
+  - name: Code Editor
+    file: code-editor.adoc
+  - name: Visual Editor
+    file: visual-editor.adoc
+  - name: Custom Build Policy Examples
+    file: custom-build-policy-examples.adoc
+- name: Setup IaC Tag and Trace
+  file: iac-tag-and-trace.adoc
+- name: Setup Drift Detection
+  file: drift-detection.adoc
+- name: Software Composition Analysis (SCA)
+  dir: software-composition-analysis
+  topics:
+  - name: Software Composition Analysis (SCA)
+    file: software-composition-analysis.adoc
+  - name: License Compliance in Software Composition Analysis (SCA)
+    file: license-compliance-in-sca.adoc
+  - name: Supported Package Managers
+    file: supported-package-managers.adoc
+- name: Secrets Scanning
+  file: secrets-scanning.adoc
+- name: Configure Application Security
+  dir: configure-application-security
+  topics:
+  - name: Configure Application Security
+    file: configure-application-security.adoc
+  - name: Enforcement (Code Reviews and PR Comments)
+    file: enforcement.adoc
+  - name: Developer Suppressions
+    file: developer-suppressions.adoc
+- name: Manage Workspace
```

Comment:
@JBakstPaloAlto- I have moved this content as a folder, as they are inter related.